### PR TITLE
Giant Bee 6969

### PR DESCRIPTION
**Pull request by romanZupancic**
;

### CODE REVIEW

File: src/main/java/Trader.java [hunk 1]
```diff
@@ -96,8 +96,8 @@ public static void main(String[] args) {
         List<Tradable> all_items = Arrays.asList(
                 new Horse(),
                 new Horse(),
-                new Horse()
-                // TODO: Add Tradable objects here!
+                new Horse(),
+                new GiantBee6969xx()
```
Comment (anadjianto) [inline src/main/java/Trader.java]: les gooo

**Comment by anadjianto** [comment]
nic

**Review by nathan-hansen — APPROVED** [review]
a work of art :O  approved!!!!@

**Comment by romanZupancic** [comment]
> a work of art :O approved!!!!@

thx boss

**Pull request merged**